File: Tidwit.Libraries/Tidwit.Libraries/Tidwit.Libraries.Domain/Entities/Library/Library.cs


namespace Tidwit.Libraries.Domain.Entities.Library
{
    using System.ComponentModel.DataAnnotations.Schema;
    using System;


    [Table("users")]
    public class Library : EntityBase
    {
        public string Description { set; get; }
        public DateTime PublishDate { set; get; }
        public string Language { set; get; }
        public bool IsPublic { set; get; }
        public string ParentLibrary { set; get; }
        public string UrlImage { set; get; }
        public string FilePath { set; get; }
    }
}

File: Tidwit.Libraries/Tidwit.Libraries/Tidwit.Libraries.UI/Libraries/src/app/Libraries/libraries/models/library.ts
export class Library {
    name:string;
    description:string;
    publishDate:Date;
    language:string;
    isPublic:boolean;
    parentLibrary:string;
    urlImage:string;
    id:number;
    filePath: string;

    constructor(){
      this.name= "";
      this.description= "";
      this.language= "";
      this.isPublic = false;
      this.parentLibrary ="";
      this.id=0;
    }
  }